mediathread/assetmgr/models.py

```diff
@@ -339,3 +339,22 @@ def __unicode__(self):
 
     class Meta:
         ordering = ['title']
+
+
+class JuxtaposeAsset(models.Model):
+    added = models.DateTimeField(auto_now_add=True)
+    modified = models.DateTimeField(auto_now=True)
+    author = models.ForeignKey(User)
+    course = models.ForeignKey(Course)
+    title = models.TextField(blank=True, null=True)
+    spine = models.ForeignKey(Source, blank=True, null=True)
+
+
+class JuxtaposeItem(models.Model):
```

Comment:
This object is intended to represent an item on one of the Juxtapose tool's tracks. I've split this out into `JuxtaposeMediaItem` and `JuxtaposeTextItem`, since they are different enough that I think that makes sense. I'm planning on renaming the 'aux' track to the 'media' track in Juxtapose.